src/capitalsFirst.js

```diff
@@ -21,7 +21,18 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let sortWords = [];
+  str.split(' ').forEach(word => {
+    if (word[0].charCodeAt() > 64 && word[0].charCodeAt() < 91) {
+      sortWords.push(word);
+    }
+  });
+  str.split(' ').forEach(word => {
```

Comment:
Will be better to create two arrays earlier, iterate once and concat them(using destruction)

Reply:
**iterate once**